csclient/params/params.go

```diff
@@ -449,3 +449,24 @@ type SetAuthCookie struct {
 	// Macaroons holds a slice of macaroons.
 	Macaroons macaroon.Slice
 }
+
+// UploadIdResponse holds the upload id response from a POST request to /upload.
+type UploadIdResponse struct {
+	UploadId string
+}
+
+// Parts holds the list of part for an upload to be completed.
+type Parts struct {
+	Parts []Part
+}
+
+// Part represents one part of a multipart blob.
+type Part struct {
+	Hash string
+}
+
+// FinishUploadResponse holds the response to a put /upload/upload-id/part-number.
```

Comment:
// FinishUploadResponse holds the response to a put /upload/upload-id/part-number request.
?